collectors/channels_collector.go

```diff
@@ -379,6 +408,33 @@ func (c *ChannelsCollector) Collect(ch chan<- prometheus.Metric) {
 		"waiting_close",
 	)
 
+	for _, forceClose := range pendingChannelsResp.PendingForceClose {
+		// Labels are: "chan_id", "status", "initiator", "peer",
+		// "blocks_until_maturity". We'll use status to hold the anchor
+		// state.
+		ch <- prometheus.MustNewConstMetric(
+			c.pendingForceCloseBalanceDesc, prometheus.GaugeValue,
+			float64(forceClose.RecoveredBalance),
+			forceClose.ChannelPoint.String(),
+			anchorStateToString(forceClose.AnchorState),
+			forceClose.ChannelInitiator.String(),
+			forceClose.PubKeyBytes.String(),
+			fmt.Sprintf("%d", forceClose.BlocksUntilMaturity),
```

Comment:
Not super happy about this but haven't found a way to add a non text label. Not a huge deal as we can still track amount changes and display maturity in a table view but would be nice to be able to track it as a number in the same metric.